extensions/wikia/FacebookTags/FacebookTagParser.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class FacebookTagParser {
+	/** @var string $tagName */
+	private $tagName;
+
+	public function __construct( string $tagName ) {
+		if ( isset( FacebookTagConstants::TAG_FALLBACKS[$tagName] ) ) {
+			$this->tagName = FacebookTagConstants::TAG_FALLBACKS[$tagName];
+			return;
+		}
+
+		$this->tagName = $tagName;
+	}
+
+	public function __invoke( ...$args ) {
+		/**
+		 * @var string $contents
+		 * @var array $attributes
+		 * @var Parser $parser
+		 * @var PPFrame $frame
+		 */
+		list( $contents, $attributes, $parser, $frame ) = $args;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$contents" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$parser" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$frame" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)